### PR TITLE
added test and changed in schema-file

### PR DESCRIPTION
**Pull request by johanneslarssonYE**

### CODE REVIEW

File: models/product_explorer/pe_schema.yml [hunk 1]
```diff
@@ -1,5 +1,66 @@
...
+          - not_null
+          - columnsexists:
+              to: source("transformed_pe", "mc_fields")
+              field: offer_id  
+                    
+    - name: brand
+      tests:
+          - columnsexists:
+              to: source("transformed_pe", "mc_fields")
+              field: brand
```
Comment (martenlindblad) [inline models/product_explorer/pe_schema.yml:18]: can we fold all the columnsexists to the same list?

Comment (johanneslarssonYE) [reply models/product_explorer/pe_schema.yml:18]: that's not possible

File: models/product_explorer/pe_schema.yml [hunk 2]
```diff
@@ -1,5 +1,66 @@
 version: 2
 
+models:
+ - name: product_explorer_fields
+   columns:
+    - name: product_sku
+      tests:
+          - unique
+          - not_null
+          - columnsexists:
```
Comment (martenlindblad) [inline models/product_explorer/pe_schema.yml:10]: I think I understand the test process. Really good.  But could we maybe rename the fields? Or are they fixed?
I don't really understand what `to` does in this context.

**Review by martenlindblad — CHANGES_REQUESTED** [review]

File: models/product_explorer/pe_schema.yml [hunk 3]
```diff
@@ -1,5 +1,66 @@
...
+            field: category_description
+
+    - name: weight
+      tests:
+        - columnsexists:
+            to: source("transformed_pe", "erp_fields")
+            field: weight         
+          
+
+
```
Comment (martenlindblad) [inline models/product_explorer/pe_schema.yml:63]: database: royaldesign-311208 should not be here please :)

**Review by martenlindblad — CHANGES_REQUESTED** [review]

File: models/product_explorer/pe_schema.yml (hunk 3 above)
Comment (martenlindblad) [reply models/product_explorer/pe_schema.yml:63]: Will create new issue for this issue

**Pull request merged**